src/Microsoft.Win32.Primitives/tests/project.json

```diff
@@ -5,7 +5,8 @@
     "xunit": "2.0.0-beta5-build2785",
     "xunit.abstractions.netcore": "1.0.0-prerelease",
     "xunit.assert": "2.0.0-beta5-build2785",
-    "xunit.core.netcore": "1.0.1-prerelease"
+    "xunit.core.netcore": "1.0.1-prerelease",
+    "xunit.netcore.extensions": "1.0.0-prerelease"
```

Comment:
Fixed, the * indicates newest version, so that we don't have to keep updating this with every new release of buildtools.